From ea57172204caa2e5dfde196cb80a36cb5449b5b7 Mon Sep 17 00:00:00 2001
From: "Wang, Long" <long1.wang@intel.com>
Date: Tue, 19 Mar 2019 11:02:49 +0800
Subject: [PATCH] support mock build

Signed-off-by: Wang, Long <long1.wang@intel.com>
---
 mockbuild/Makefile | 5 +++++++
 1 file changed, 5 insertions(+)
 create mode 100644 mockbuild/Makefile

diff --git a/mockbuild/Makefile b/mockbuild/Makefile new file mode
100644 index 0000000..be79604
--- /dev/null
+++ b/mockbuild/Makefile
@@ -0,0 +1,12 @@
+install:
+	mkdir -p $(INSTALLROOT)/usr/bin/
+	mkdir -p $(INSTALLROOT)/usr/lib64/
+	find ../src/lib -name *.so* -exec sh -c 'install {} $(INSTALLROOT)/usr/lib64/' \;
+	mkdir -p $(INSTALLROOT)/usr/lib64/pkgconfig/
+	mkdir -p $(INSTALLROOT)/usr/include/ndpi/
+	mkdir -p $(INSTALLROOT)/usr/share/doc/ndpi/
+	find ../src/lib -name *.so* -exec sh -c 'cp -a {} $(INSTALLROOT)/usr/lib64/' \;
+	find ../example -executable -exec sh -c 'install {} $(INSTALLROOT)/usr/bin/' \;
+	find .. -name *.pc -exec sh -c 'install {} $(INSTALLROOT)/usr/lib64/pkgconfig/' \;
+	find ../src/include -name *.h -exec sh -c 'install {} $(INSTALLROOT)/usr/include/ndpi/' \;
+	find ../doc * -exec sh -c 'install {} $(INSTALLROOT)/usr/share/doc/ndpi/' \;
--
2.20.1

